service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledResourceServiceTest.java

```diff
@@ -165,7 +176,7 @@ public void resetFlightDebugInfo() {
 
   @Test
   @DisabledIfEnvironmentVariable(named = "TEST_ENV", matches = BUFFER_SERVICE_DISABLED_ENVS_REG_EX)
-  public void createAiNotebookInstanceDo() throws Exception {
+  void createAiNotebookInstanceDo() throws Exception {
```

Comment:
Does visibility matter for tests? I always make them public

Reply:
As of JUnit 5, not really. The `@Test` annotation documentation just states that they can't be `private` or `static` and must not have a return type.
I have a very slight style preference for package-private since it's less typing, but that was definitely outweighed by refactoring this file.